pom.xml

```diff
@@ -789,6 +789,7 @@
                                         <mariadb.105.image>registry.redhat.io/rhel8/mariadb-105</mariadb.105.image>
                                         <amq-streams.image>registry.redhat.io/amq7/amq-streams-kafka-27-rhel7</amq-streams.image>
                                         <amq-streams.version>1.7.0</amq-streams.version>
+                                        <mongodb.image>docker.io/library/mongo@sha256:6f851e31a1b317c6fa681b7dad5f94c622f1c3588477f3b769579dc5462ee659</mongodb.image>
```

Comment:
Why doesn't the default `<mongodb.image>docker.io/library/mongo:5.0</mongodb.image>` work?
The intention to move to the official image instead of bitnami was to have OOTB experience on arm.

Also a note how to figure out the right image / sha256 would be handy.

Reply:
> Why doesn't the default <mongodb.image>docker.io/library/mongo:5.0</mongodb.image> definition work?

It works, but it is image build for platform amd64, which is not what we want to test

> The intention to move to the official image instead of bitnami was to have OOTB experience on arm.

move to the official image was so that we can use ARM64 platform specific image, please note that Docker images are built for different platforms. You can try `docker manifest inspect mongo:5.0` and see what are possibilities for different platforms

> Also a note how to figure out the right image / sha256 would be handy.

I wrote it into this description. I can write it into comment (well, I'll do it for sake of having this approved), but I think if someone want to change something, he should know what he is doing (or learn it from history).

Reply:
Added comment how to choose correct image digest.

Reply:
Sorry, I don't see in the PR's description, I had planned to write it there.